app/Http/Controllers/Auth/ArticleController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\UpdateArticleRequest;
+use App\Models\Article;
+use App\Models\Category;
+use App\Models\Tag;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class ArticleController extends Controller
+{
+    public function index(): View
+    {
+        $articles = Article::latest()->paginate(10);
+
+        return view('auth.articles.index', compact('articles'));
+    }
+
+    public function show(Article $article): View
+    {
+        return view('auth.articles.show', compact('article'));
+    }
+
+    public function create(): View
+    {
+        return view('auth.articles.create');
+    }
+
+    /**
+     * @throws \Illuminate\Validation\ValidationException
+     */
+    public function store(Request $request): RedirectResponse
+    {
+        $request->validate([
```

Comment:
In update you use Form Requests but here you validate manually in controller. Use one option for consistency. And with form requests you can use `$request->validated()`